src/data.js

```diff
@@ -19,50 +19,73 @@ const state = {
     correct: 0,
     // how do you want to measure score?
     //  that's up to you!
-    score: 0
+    score: 0,
   },
   // the questions in the quiz
   questions: [
     {
       text: 'What are the different ways to declare a JS variable?',
-      answers: [
-        'constant, let, variable',
-        'var, const, let, function',
-        'var, let, const'
-      ],
+      answers: ['constant, let, variable', 'var, const, let, function', 'var, let, const'],
       correct: 2,
       selected: -1,
       links: [
         {
           text: 'javascript.info',
-          href: 'https://javascript.info/variables'
-        },
-        {
-          text: 'Tyler McGinnis',
-          href: 'https://ui.dev/var-let-const/'
+          href: 'https://javascript.info/variables',
         },
-      ]
+      ],
     },
     {
       text: 'What does `typeof` do?',
       answers: [
         'changes the type of a primitive value',
         'returns a string describing the type of a value',
         'determines if a value is primitive',
-        'can tell the difference between arrays and objects',
       ],
       correct: 1,
       selected: -1,
       links: [
         {
           text: 'javascript.info',
-          href: 'https://javascript.info/types#type-typeof'
+          href: 'https://javascript.info/types#type-typeof',
         },
+      ],
+    },
+    {
+      text: 'How can you create an object in JavaScript?',
+      answers: ['Using curly brackets{}', 'Using square brackets[]', 'Using the magic word Object'],
+      correct: 0,
+      selected: -1,
+      links: [
         {
-          text: 'MDN',
-          href: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof'
+          text: 'javascript.info',
+          href: 'https://javascript.info/object',
         },
-      ]
+      ],
+    },
+    {
+      text: 'Who is the best teacher for web development?',
+      answers: ['Evan Cole', 'Mosh Hamedani', 'Wes Bos', 'Brad - Traversy Media'],
```

Comment:
I think, we should keep 3 options of answers for all questions. We have only 3 buttons in index.html

Reply:
I fix this problem @aleks2407

Reply:
thank you!